models/topic-model.js

```diff
@@ -29,4 +29,36 @@ const selectUsers = () => {
   });
 };
 
-module.exports = { selectTopics, selectUsers, selectArticles, selectArticleById };
+const setArticleById = (inc_votes, article_id) => {
+  if (!inc_votes || typeof parseInt(article_id) != "number") {
```

Comment:
please use !== over !=. We will avoid type coercion this way. Always stick to stricy equality.